boards/FSM/Src/App/App_AcceleratorPedals.c

```diff
@@ -170,59 +170,51 @@ float App_AcceleratorPedals_GetSecondaryPedalPercentage(const struct Accelerator
-    //                                             CANMSGS_FSM_MOTOR_SHUTDOWN_ERRORS_PAPPS_OCSC_IS_ACTIVE_FALSE_CHOICE;
-    // App_CanTx_SetPeriodicSignal_PAPPS_OCSC_IS_ACTIVE(can_tx,
-    // CANMSGS_FSM_MOTOR_SHUTDOWN_ERRORS_PAPPS_ALARM_IS_ACTIVE);
+    uint8_t PappsAlarmEnum =
+        papp_signal_state == SIGNAL_STATE_ACTIVE ? PAPPS_OCSC_IS_ACTIVE_TRUE_CHOICE : PAPPS_OCSC_IS_ACTIVE_FALSE_CHOICE;
+    App_CanTx_FSM_MotorShutdownErrors_PappsOCSCIsActive_Set(PappsAlarmEnum);
+
     const bool  secondary_pedal_ocsc = accelerator_pedals->secondary_pedal_OCSC();
     SignalState sapp_signal_state =
         App_SharedSignal_Update(accelerator_pedals->sapp_alarm_signal, secondary_pedal_ocsc, !secondary_pedal_ocsc);
-    // TODO: JSONCAN ->
-    // uint8_t CANMSGS_FSM_MOTOR_SHUTDOWN_ERRORS_SAPPS_ALARM_IS_ACTIVE =
-    //    sapp_signal_state == SIGNAL_STATE_ACTIVE ? CANMSGS_FSM_MOTOR_SHUTDOWN_ERRORS_SAPPS_OCSC_IS_ACTIVE_TRUE_CHOICE
-    //                                             :
-    //                                             CANMSGS_FSM_MOTOR_SHUTDOWN_ERRORS_SAPPS_OCSC_IS_ACTIVE_FALSE_CHOICE;
-    // App_CanTx_SetPeriodicSignal_SAPPS_OCSC_IS_ACTIVE(can_tx,
-    // CANMSGS_FSM_MOTOR_SHUTDOWN_ERRORS_SAPPS_ALARM_IS_ACTIVE);
+    uint8_t SappsAlarmEnum =
+        sapp_signal_state == SIGNAL_STATE_ACTIVE ? SAPPS_OCSC_IS_ACTIVE_TRUE_CHOICE : SAPPS_OCSC_IS_ACTIVE_FALSE_CHOICE;
+    App_CanTx_FSM_MotorShutdownErrors_SappsOCSCIsActive_Set(SappsAlarmEnum);
+
     // torque 0
     if (papp_signal_state == SIGNAL_STATE_ACTIVE || sapp_signal_state == SIGNAL_STATE_ACTIVE)
     {
-        // TODO: JSONCAN ->
-        // App_CanTx_SetPeriodicSignal_MAPPED_PEDAL_PERCENTAGE(can_tx, 0.0f);
-        // App_CanTx_SetPeriodicSignal_SAPPS_MAPPED_PEDAL_PERCENTAGE(can_tx, 0.0f);
+        App_CanTx_FSM_Apps_PappsMappedPedalPercentage_Set(0.0f);
+        App_CanTx_FSM_Apps_SappsMappedPedalPercentage_Set(0.0f);
     }
 
     // Primary Secondary Accelerator Agreement (Inaccurate data)
     const float papp_sapp_diff =
         accelerator_pedals->get_primary_pedal_percent() - accelerator_pedals->get_secondary_pedal_percent();
     SignalState app_agreement_signal_state = App_SharedSignal_Update(
         accelerator_pedals->app_agreement_signal, (papp_sapp_diff) > 10.f, (papp_sapp_diff) <= 10.f);
-    // TODO: JSONCAN ->
-    // uint8_t CANMSGS_FSM_MOTOR_SHUTDOWN_ERRORS_APPS_HAS_DISAGREEMENT =
-    //    app_agreement_signal_state == SIGNAL_STATE_ACTIVE
-    //        ? CANMSGS_FSM_MOTOR_SHUTDOWN_ERRORS_APPS_HAS_DISAGREEMENT_TRUE_CHOICE
-    //        : CANMSGS_FSM_MOTOR_SHUTDOWN_ERRORS_APPS_HAS_DISAGREEMENT_FALSE_CHOICE;
-    // App_CanTx_SetPeriodicSignal_APPS_HAS_DISAGREEMENT(can_tx,
-    // CANMSGS_FSM_MOTOR_SHUTDOWN_ERRORS_APPS_HAS_DISAGREEMENT); if (app_agreement_signal_state == SIGNAL_STATE_ACTIVE)
-    //    App_CanTx_SetPeriodicSignal_MAPPED_PEDAL_PERCENTAGE(can_tx, 0.0f);
+    uint8_t AppsDisagreementEnum = app_agreement_signal_state == SIGNAL_STATE_ACTIVE
```

Comment:
nit: varialbes names use `snake_case`, so this should be `apps_disagreement_enum`